helm/event-exporter-app/templates/deployment.yaml

```diff
@@ -1,11 +1,14 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Values.name }}
+  name: {{ include "event-exporter.fullname" . }}
   namespace: {{ .Release.Namespace }}
-  labels:
-    {{- include "labels.common" . | nindent 4 }}
+  labels: {{ include "event-exporter.labels" . | nindent 4 }}
 spec:
+  selector:
+    matchLabels:
+      app: event-exporter-app
+      version: v1
```

Comment:
Why not use the `event-exporter.selectorLabels` helper? Why are these different?

Reply:
Because you cannot change the selector match labels of an existing Deployment. They are immutable. And since someone decided to just hard-code to this, we now need to keep them.

Only thing I can do is extending the pod template's labels and then use the `event-exporter.selectorLabels` in the NetworkPolicy. But sadly not in the Deployment, at least not as long as I don't remove the whole app from all MCs and re-create it. Like... removing the app from the app collection and adding it again.

Reply:
You know what? You were right! I took another look into the code and we could use the selector labels. I just changed this, new commit incoming.

Reply:
I updated the helper. Before we were already using the chart name, not the release name for `app:`. I mean, ok, it's still not perfect, but at least it's the same everywhere now.